src/EnergyPlus/api/runtime.py

```diff
@@ -189,6 +191,18 @@ def run_energyplus(self, state: c_void_p, command_line_args: List[Union[str, byt
     # def stop_simulation(self, state: c_void_p) -> None:
     #     pass
 
+    def set_console_output_status(self, state, print_output: bool) -> None:
```

Comment:
API function with minimal wrapping in the Python API.  For Python we use a simple boolean to toggle the console output on or off.